sdk/go/pulumi-language-go/main_test.go

```diff
@@ -456,92 +454,3 @@ func testPluginsAndDependencies(t *testing.T, progDir string) {
 		}, gotDeps)
 	})
 }
-
```

Comment:
These are now covered by the smoke tests in tests/smoke_test.go, and will be more properly covered by matrix testing.